pkg/pillar/pubsub/publish.go

```diff
@@ -35,8 +35,8 @@ type Notify struct{}
 //  foo := p1.Get(key)
 //  fooAll := p1.GetAll()
 
-// PublicationImpl - Publication Implementation. The main structure that implements
-//  Publication interface.
+// PublicationImpl - NewPublication Implementation. The main structure that implements
+//  NewPublication interface.
```

Comment:
Same as before, revert comment. This implements `Publication interface`, the interface, not `NewPublication`, the func call.